### PR TITLE
Configure 'prettify-updated' script

### PR DESCRIPTION
**Pull request by medikoo**
Additionally improve .editorconfig, so it allows trailing whitespace in markdown

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -10,3 +10,6 @@ insert_final_newline = true
 indent_size = 2
 indent_style = space
 trim_trailing_whitespace = true
+
+[*.md]
+trim_trailing_whitespace = false
```
Comment (pmuens) [inline .editorconfig]: Why don't we trim whitespaces for `.md` files?

Comment (medikoo) [reply .editorconfig]: In a paragraph two blank spaces at end of the line could be used to force new line -> https://commonmark.org/help/tutorial/03-paragraphs.html

Comment (pmuens) [reply .editorconfig]: TIL :+1:

Comment (medikoo) [reply .editorconfig]: In other cases, I believe Prettier will trim unnecessary space, so style wise it's safe

**Pull request merged**